### PR TITLE
Update panorama-upgrade.html.md.erb

### PR DESCRIPTION
**Pull request by madjava**
Updated info about current version detail.

### CODE REVIEW

File: source/network/panorama-upgrade.html.md.erb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 weight: 100
-last_reviewed_on: 2024-01-24
-review_in: 4 months
+last_reviewed_on: 2024-04-14
+review_in: 3 months
 ---
 # <%= current_page.data.title %>
 
-The current Panorama, Log Collectors version is 11.0.2 and Firewall version is `10.1.5-h1`, `h` stands for Hotfix.
+The current Panorama, Log Collectors and Firewall version version is `11.0.2-h3`, `h` stands for Hotfix.
```
Comment (thomast1906) [inline source/network/panorama-upgrade.html.md.erb:9]: Do we want to having to keep this up to date or just link to where you can find latest version?

Comment (madjava) [reply source/network/panorama-upgrade.html.md.erb:9]: We are not at the edge and usually pick the next best version for the series we currently are on. Not sure what other best way to keep this updated than to update this doc when we do the next update as well.

Comment (madjava) [reply source/network/panorama-upgrade.html.md.erb:9]: > Do we want to having to keep this up to date or just link to where you can find latest version?

would update typo 👍🏾

**Review by thomast1906 — APPROVED** [review]

File: source/network/panorama-upgrade.html.md.erb (hunk 1 above)
Comment (thomast1906) [reply source/network/panorama-upgrade.html.md.erb]: sorry, mean't link to where a person would check what version we run. 

Have approved for now, was just a think

**Pull request merged**